Makefile.am: link against libpthread for atftp

atftp client needs to link against libpthread as static builds do not
work otherwise.

Signed-off-by: Ryan Barnett <ryan.barnett@rockwellcollins.com>
---
 Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index e59592f..200829a 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -28,7 +28,7 @@ noinst_HEADERS   = argz.h logger.h options.h stats.h tftp.h tftp_def.h tftp_io.h
 		   tftpd.h tftpd_pcre.h tftpd_mtftp.h
 
 bin_PROGRAMS     = atftp
-atftp_LDADD      = $(LIBTERMCAP) $(LIBREADLINE)
+atftp_LDADD      = $(LIBTERMCAP) $(LIBREADLINE) $(LIBPTHREAD)
 atftp_SOURCES    = tftp.c tftp_io.c logger.c options.c tftp_def.c tftp_file.c \
 		   argz.c tftp_mtftp.c
 
-- 
1.9.1

